.github/workflows/linkcheck.yml

```diff
@@ -26,6 +26,7 @@ jobs:
       run: |
         python -V
         python -m pip install --upgrade pip
+        pip install -r tests/requirements.txt
         pip install "sphinx==${{ env.sphinx-version }}"
         pip install "jinja2<3.1"
```

Comment:
Should jinja2<3.1 also be added to tests/requirements.txt instead of being installed on a separate line?

Reply:
`jinja2<3.1` is only need when using older versions of `sphinx`, e.g. here we use `sphinx-version: '5.*'`. That is the reason why I have not included it in `tests/requirements.txt`. But I agree, there might be smarter ways to specify the dependencies compared to current approach.